src/jvm/main/org/jetbrains/kotlinx/lincheck/Lincheck.kt

```diff
@@ -32,8 +32,9 @@ object Lincheck {
     @JvmStatic
     fun runConcurrentTest(
         invocations: Int = DEFAULT_INVOCATIONS_COUNT,
-        block: Runnable
-    ) {
+        analyzeStdLib: Boolean = false,
```

Comment:
Please do not change the API, add a system property instead.

Reply:
After a discussion, decided to leave this option, but make the API internal, only expose publicly the version without additional options.